pywbem/cim_types.py

```diff
@@ -855,38 +860,42 @@ def type_from_name(type_name):
 
 def atomic_to_cim_xml(obj):
     """
-    Convert a value of an atomic scalar CIM data type to a CIM-XML string and
-    return that string.
+    Convert an atomic scalar value to a CIM-XML string and return that string.
 
-    TODO: Verify whether we can change this function to raise a ValueError in
-    case the value is not CIM typed.
+    The returned CIM-XML string is ready for use as the text of a CIM-XML
+    'VALUE' element.
 
     Parameters:
 
-      obj (atomic scalar CIM typed value):
-        The CIM typed value, including `None`. Must be a scalar (not an array).
-        Must be an atomic type (i.e. those listed in :ref:`CIM data types`,
-        except any :ref:`CIM objects`).
+      obj (:term:`CIM data type`, :term:`number`, :class:`py:datetime`):
```

Comment:
Suggest build the word atomic back in just to help explain why we call it atomic... in the function call.  To be hones I am not sure why we use the word atomic

Reply:
Me neither. I think we inherited that as part of the public API.
DONE.